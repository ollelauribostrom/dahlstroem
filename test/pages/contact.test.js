import React from 'react';
import { shallow } from 'enzyme';

import ContactPage from '../../src/pages/contact';

describe('Tests for Contact page', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<ContactPage location={{ pathname: '/test' }} />);
    expect(wrapper).toMatchSnapshot();
  });
});
