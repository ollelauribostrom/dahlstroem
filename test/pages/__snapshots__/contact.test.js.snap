// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for Contact page renders correctly 1`] = `
<Layout
  color={
    Object {
      "hex": "#000",
    }
  }
  menuBackground={
    Object {
      "hex": "#000",
    }
  }
  menuColor={
    Object {
      "hex": "#fff",
    }
  }
  path="/test"
>
  <SEO
    keywords={
      Array [
        "contact",
      ]
    }
    lang="en"
    meta={Array []}
    title="Contact"
  />
  <div
    style={Object {}}
  >
    <h3
      style={
        Object {
          "color": "#6e297a",
          "fontSize": "18px",
          "marginBottom": "10px",
        }
      }
    >
      Contact
    </h3>
    <span
      style={
        Object {
          "display": "block",
        }
      }
    >
      Felix Dahlstr√∂m Persson
    </span>
    <span
      style={
        Object {
          "display": "block",
        }
      }
    >
      Thorvald Meyers Gate 71A
    </span>
    <span
      style={
        Object {
          "display": "block",
        }
      }
    >
      0552 Oslo, Norway
    </span>
    <span
      style={
        Object {
          "display": "block",
        }
      }
    >
      dahlstrom.persson@gmail.com
    </span>
  </div>
</Layout>
`;
